dataflows-inline/reddit-streamer/cleancon

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
@sehz there have been many requests to create connectors that pull from different websites, this is one of those tasks as well.